packages/lesswrong/components/posts/AddToCalendar/makeUrls.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * This file is copied from https://github.com/CultureHQ/add-to-calendar/blob/master/src/makeUrls.ts
+ * (I deleted Yahoo since no one uses that)
```

Comment:
Add their license:

https://github.com/CultureHQ/add-to-calendar/blob/master/LICENSE